src/releasers/python.ts

```diff
@@ -20,8 +20,16 @@ import {Changelog} from '../updaters/changelog';
 // Python specific.
 import {SetupPy} from '../updaters/python/setup-py';
 import {SetupCfg} from '../updaters/python/setup-cfg';
-import {VersionPy} from '../updaters/python/version-py';
+import {PyBasedFile} from '../updaters/python/py-based-file';
```

Comment:
I'm not a big fan of the new name `PyBasedFile`.
It makes some sense in a context of `pyproject.toml` with the `name` field, but it doesn't make sense for the original `version.py` case.

Can we keep the original name and add a comment about `__init__.py` case?
Or maybe `python-file-with-version.ts` and `PythonFileWithVersion`?